examples/tv-app/android/java/ContentAppCommandDelegate.cpp

```diff
@@ -110,9 +111,8 @@ void ContentAppCommandDelegate::InvokeCommand(CommandHandlerInterface::HandlerCo
 void ContentAppCommandDelegate::FormatResponseData(CommandHandlerInterface::HandlerContext & handlerContext, const char * response)
 {
     Json::Reader reader;
-    Json::Value resJson;
-    reader.parse(response, resJson);
-    Json::Value value = resJson["value"];
+    Json::Value value;
+    reader.parse(response, value);
```

Comment:
parse returns a bool for success/failure. Please handle failure.

Reply:
Added failure handling for parsing.